sov-esdt-safe/tests/setup.rs

```diff
@@ -16,26 +20,10 @@ use proxies::{
 };
 use sov_esdt_safe::SovEsdtSafe;
 
-pub const CONTRACT_ADDRESS: TestSCAddress = TestSCAddress::new("sc");
 pub const SOV_ESDT_SAFE_CODE_PATH: MxscPath = MxscPath::new("output/to-sovereign.mxsc.json");
```

Comment:
`output/to-sovereign.mxsc.json`
How is this working? The contract generates file named `sov-esdt-safe.mxsc.json`